### DISCUSSION TITLE
Image generation based on text & image prompt but regardless of uploaded photo clothes

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2748] [General]
It seems the model also picks up on what clothes the user is wearing in the photo, however users may upload poor photos of themselves which may not match the clothes they want. Perhaps there's something we can do here. Maybe matter of text prompt. Appreciate if lettting me know. 
Hope to hear from expert help. 
Thank you.

**Comment by Rogala** [comment]
If you use ImagePrompt-Advanced_ImagePrompt, then yes, it takes a part of the original image
If you use ImagePrompt-Advanced_PyraCanny, then only the basic outline of the elements in the image is taken, it doesn't care what is drawn there.
If you use ImagePrompt-Advanced_CPDS, only the color mask of the elements in the image is taken, it doesn't care what is drawn there.
If you use ImagePrompt-Advanced_FaceSwap, only the face from the image is taken into account.
You can choose what you want, but take into account Stop At (how long to take data from the image) and Weight (how the data from the image affects the new version).

**Reply by ILikeToasters** [reply]
Actually I would add that FaceSwap will also take into account other things like jewelry and background.
